lib/generators/shopify_app/add_after_authenticate_job/add_after_authenticate_job_generator.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails/generators/base'
+
+module ShopifyApp
+  module Generators
+    class AddAfterAuthenticateJobGenerator < Rails::Generators::Base
+      source_root File.expand_path('../templates', __FILE__)
+
+      hook_for :test_framework, as: :job, in: :rails do |instance, generator|
+        instance.invoke generator, [ instance.send(:job_file_name) ]
+      end
+
+      def init_after_authenticate_config
+        initializer = load_initializer
+        return if initializer.include?("config.webhooks")
+
+        inject_into_file(
+          'config/initializers/shopify_app.rb',
+          "  config.after_authenticate_job = { job: Shopify::AfterAuthenticateJob, inline: false }\n",
```

Comment:
oh I see - its not really a default argument the way I was thinking of it. Maybe it is better to have here since otherwise it would only live in documentation. What are your thoughts @Hammadk